synapse/eventbus.py

```diff
@@ -124,6 +124,41 @@ def unlink(self, func):
         if func in self._syn_links:
             self._syn_links.remove(func)
 
+    def preemptive_on(self, name, func, **filts):
```

Comment:
not a big fan of this idea because of the slippery slope effect ( code getting into fights about preemption or arbitrary event ordering.  "no, I'm first", "no, me me!". ) and the fact that it feels like a one-off.  Any way we can do without it?